downloader/oci_getter.go

```diff
@@ -31,23 +33,29 @@ func (g *OCIGetter) Get(path string, u *url.URL) error {
 		return fmt.Errorf("make policy directory: %w", err)
 	}
 
-	cli, err := auth.NewClient()
+	cli, err := dockerauth.NewClient()
 	if err != nil {
 		return fmt.Errorf("new auth client: %w", err)
 	}
 
-	resolver, err := cli.Resolver(ctx, http.DefaultClient, false)
+	opts := []auth.ResolverOption{auth.WithResolverClient(http.DefaultClient)}
+	resolver, err := cli.ResolverWithOpts(opts...)
+	if err != nil {
+		return fmt.Errorf("docker resolver: %w", err)
+	}
+
+	registry := content.Registry{Resolver: resolver}
 	if err != nil {
-		return fmt.Errorf("new resolver: %w", err)
+		return fmt.Errorf("creating registry: %w", err)
 	}
```

Comment:
It doesn't look like the previous line's assignment for `registry` has an error value, so this is reusing the same error from the `cli.ResolverWithOpts()` call above. I think this can just be removed.